src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -49,7 +50,8 @@ const int kMaxReorderWaitSeconds = 30;
 // Interface for testing.
 class CoreWorkerDirectActorTaskSubmitterInterface {
  public:
-  virtual void AddActorQueueIfNotExists(const ActorID &actor_id) = 0;
+  virtual void AddActorQueueIfNotExists(const ActorID &actor_id,
```

Comment:
@rkooo567 @stephanie-wang is there any reason we can't lazily create it when SubmitTask is called?